plugins/chat/assets/javascripts/discourse/components/chat-footer.gjs

```diff
@@ -4,6 +4,7 @@ import DButton from "discourse/components/d-button";
 import concatClass from "discourse/helpers/concat-class";
 import i18n from "discourse-common/helpers/i18n";
 import eq from "truth-helpers/helpers/eq";
+import ChatFooterUnreadIndicator from "discourse/plugins/chat/discourse/components/chat/footer/unread-indicator";
```

Comment:
Just a detail, but Im not trying to importe this kind of things as: `FooterUnreadIndicator` we don't really care about "Chat" here.

Reply:
Ah okay thanks. I saw we use Chat as the prefix in so many imports, so I was following suit here. Good to know that we are trying to remove this so I can avoid in future.